apps/frontpage/scripts/getLocalDocs.ts

```diff
@@ -2,9 +2,7 @@ import path from 'path';
 import fs from 'fs-extra';
 import fetch from 'node-fetch';
 import tar from 'tar';
-// TODO: Figure out why we can't use an absolute import here
-// import { DocsVersion, docsVersions } from "../docs-versions";
-import { DocsVersion, docsVersions } from './docs-versions';
+import { DocsVersion, docsVersions } from '@utils';
```

Comment:
@kylegach This did work? On my side it did not. Not sure why.

Reply:
Yes, I got it to work. Here's the key, in `apps/frontpage/project.json`:

```diff
"fetch-docs": {
-  "command": "tsx apps/frontpage/scripts/getLocalDocs.ts"
+  "command": "tsx --tsconfig apps/frontpage/tsconfig.json apps/frontpage/scripts/getLocalDocs.ts"
},
```

This just helped me realize that I should add that to the other `tsx` usage, too.